recipe/meta.yaml

```diff
@@ -1,20 +1,19 @@
 {% set version = "0.3.2" %}
-{% set sha256 = "c20239fe3dcf6bd2ac957395b712f52e630c83a38d5347e7a3e6dce718ad4fe1" %}
 
 package:
   name: mtspec
   version: {{ version }}
 
 
 source:
-  fn: mtspec-{{ version }}.zip
   url: https://pypi.io/packages/source/m/mtspec/mtspec-{{ version }}.zip
-  sha256: {{ sha256 }}
+  sha256: c20239fe3dcf6bd2ac957395b712f52e630c83a38d5347e7a3e6dce718ad4fe1
 
 build:
-  number: 2
+  number: 3
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
   skip: True  # [win]
+  skip: True  # [py2k]
```

Comment:
It is probably possible to fix legacy Python here but I don't think it is worth it.